amazon/dynamodb/client/src/main/kotlin/org/http4k/connect/amazon/dynamodb/model/Attribute.kt

```diff
@@ -164,14 +164,21 @@ class Attribute<FINAL>(
         override fun required(name: String, description: String?): Attribute<OUT> {
             val getLens = get(name)
             val setLens = set(name)
+            val meta = Meta(true, location, paramMeta, name, description)
             return Attribute(
                 dataType,
-                Meta(true, location, paramMeta, name, description),
-                {
-                    getLens(it).firstOrNull()
-                        ?: throw LensFailure(Missing(Meta(true, location, paramMeta, name, description)), target = it)
-                },
+                meta,
+                { getLens(it).firstOrNull() ?: throw LensFailure(Missing(meta), target = it) },
                 { out: OUT, target -> setLens(listOf(out), target) })
         }
     }
 }
+
+fun <OUT> Attribute<OUT?>.toRequired(description: String? = null): Attribute<OUT> {
+    val requiredMeta = meta.copy(required = true, description = description)
```

Comment:
I wondered if we should keep the original meta `description` if no new description has been provided. I decided not to do this since the description is probably very specific for an attribute.

Reply:
The descriptions aren't really used for the dynamo attributes (they are inherited from the lens system), but we can use the old version or override it with the new if supplied :)